### PR TITLE
Adds atomic header

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds atomic header

## Additions

- Atomicizes header

## Removals

- Old header less file.

## Testing

- Visit any page, resize page, note header.

## Review

- @KimberlyMunoz 
- @sebworks 
- @jimmynotjim 

## Screenshots

screen shot 2016-01-22 at 7 06 39 pm

screen shot 2016-01-22 at 7 06 48 pm

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/templates/header.html [hunk 1]
```diff
@@ -31,21 +31,16 @@
...
+            <img class="o-header_logo-img u-js-only"
+                 src="/static/img/logo_sm-exec.svg"
+                 onerror="this.onerror=null;this.src='/static/img/logo_sm-exec.png';"
                  alt="Consumer Financial Protection Bureau"
                  width="237"
                  height="50">
             <noscript>
-                <img class="header_logo logo__no-js"
+                <img class="o-header_logo-img
+                            o-header_logo-img__no-js"
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/templates/header.html]: Why do we need the modifier here?

Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/templates/header.html]: Go point, I just carried `logo__no-js` over from the old setup. I'll check into it.

Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/templates/header.html]: Removed modifier in https://github.com/cfpb/cfgov-refresh/commit/7130a237315ae210726d0d29a0d05cd2b5b32728

**Comment by sebworks** [comment]
:+1:

**Pull request merged**